server/src/test/omtLinkProvider.test.ts

```diff
@@ -0,0 +1,114 @@
+import { expect } from 'chai';
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { assert, stub } from 'sinon';
+import { DocumentLink, Position, Range } from 'vscode-languageserver';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+import OMTLinkProvider from '../omtLinkProvider';
+import { WorkspaceLookup } from '../workspaceLookup';
+
+type Case = { i: number, line: number, start: number, end: number, target: string | undefined, should: string };
+
+describe('OMTLinkProvider', () => {
+    let linkProvider: OMTLinkProvider;
+    let lookupStub: WorkspaceLookup;
+
+    beforeEach(() => {
+        lookupStub = <WorkspaceLookup>{
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            getModulePath: (module: string) => '',
```

Comment:
it was still generating a lint exception with eslint (that is why I removed it). I can not find a way to configure the rule so that it does not with the underscore.
I will re-add an underscore because I see another part of the IDE does recognize it and it is not marked in the darker color as unused anymore.

Reply:
You could use this from the eslint rule configuration: https://eslint.org/docs/rules/no-unused-vars#argsignorepattern

Reply:
tslint I think does this by default